src/Markdig/MarkdownExtensions.cs

```diff
@@ -575,5 +589,17 @@ public static MarkdownPipelineBuilder Configure(this MarkdownPipelineBuilder pip
             }
             return pipeline;
         }
+
+        /// <summary>
+        /// Configures the string to be used for line-endings, when writing.
+        /// </summary>
+        /// <param name="pipeline">The pipeline.</param>
+        /// <param name="newLine">The string to be used for line-endings.</param>
+        /// <returns>The modified pipeline</returns>
+        public static MarkdownPipelineBuilder ConfigureNewLine(this MarkdownPipelineBuilder pipeline, string newLine)
+        {
+            pipeline.Use(new ConfigureNewLineExtension(Environment.NewLine));
```

Comment:
This should be `newLine` here instead of `Environment.NewLine`

Reply:
Good catch! At least one of us is awake! :smile: